base/base.jl

```diff
@@ -34,6 +34,11 @@ type MethodError <: Exception
     args
 end
 
+type AmbiguousError <: Exception
```

Comment:
Ambiguous or Ambiguity? Most of the other exception types are nouns not adjectives